uber/models/email.py

```diff
@@ -263,3 +263,9 @@ def fk_email(self):
     @property
     def format(self):
         return 'html' if self.is_html else 'text'
+
+    @property
+    def is_html(self):
+        return self.automated_email.is_html \
+            if self.automated_email_id and self.automated_email \
+            else super(Email, self).is_html
```

Comment:
More robust detection for `is_html`. If the `Email` is attached to an `AutomatedEmail`, then the `AutomatedEmail`'s setting is used. Otherwise fall back to the default. 

**NOTE**: this will still fail for non-automated emails.